tests/functional/s/superfluous_parens.py

```diff
@@ -1,19 +1,27 @@
 """Test the superfluous-parens warning."""
 from __future__ import print_function
-# pylint: disable=unneeded-not
+
+# pylint: disable=unneeded-not, missing-function-docstring, unnecessary-comprehension
 i = 3
-if (i == 5):  # [superfluous-parens]
+if i == 5:
```

Comment:
This one is valid and should still work after we fix the false positives.

Reply:
Shit, this is due to black auto-removing the parens and me wondering why the error should show up for `if i == 5`